crypto/bn/bn_lib.c

```diff
@@ -142,18 +142,18 @@ int BN_num_bits_word(BN_ULONG l)
 #if defined(THIRTY_TWO_BIT) || defined(SIXTY_FOUR_BIT) || defined(SIXTY_FOUR_BIT_LONG)
         if (l & 0xffff0000L) {
             if (l & 0xff000000L)
-                return (bits[(int)(l >> 24L)] + 24);
+                return bits[(int)(l >> 24L)] + 24;
```

Comment:
Missed opportunity to remove superfluous L in constant.

Reply:
`L` is a funny character, in different projects the reverse is true, `L` must be added even to `1`, if it's a long. C is not altogether "fun" :)